openssl/src/x509/mod.rs

```diff
@@ -78,6 +78,29 @@ impl X509StoreContextRef {
     pub fn error_depth(&self) -> u32 {
         unsafe { ffi::X509_STORE_CTX_get_error_depth(self.as_ptr()) as u32 }
     }
+
+    pub fn get_chain(&self) -> Option<&mut StackRef<X509Ref>> {
```

Comment:
That looked weird to me as well but the pointer returned by the OpenSSL method is not const. You think that's a mistake? I must admit that OpenSSL seems rather inconsistent with its uses of consts.